OracleWebCenterContent/dockerfiles/README.MD

```diff
@@ -0,0 +1,315 @@
+
+This option lets you mount a directory from your host to a container as volume. This volume is used to mainly store WebLogic server domain files. The volume is created at this location `/scratch/wccdocker/volumes/wccontentvolume`.
+
+To mount a host directory as a data volume, execute the below command.
+
+> The userid can be anything but it must belong to uid:guid as 1000:1000, which is same as 'oracle' user running in the container.
+
+> This ensures 'oracle' user has access to shared volume.
+
+```
+sudo mkdir -p /scratch/wccdocker/volumes/wccontent
+sudo chown 1000:1000 /scratch/wccdocker/volumes/wccontent
+```
+
+All container operations are performed as **'oracle'** user.
+
+**Note**: If a user already exists with **'-u 1000 -g 1000'** then use the same user. Or modify any existing user to have uid-gid as **'-u 1000 -g 1000'**
+
+## 3.5. Database
+You need to have a running database container or a database running on any machine. 
+The database connection details are required for creating WebCenter Content specific RCU schemas while configuring WebCenter Content domain. 
+
+The Oracle Database image can be pulled from the [Docker Store](https://store.docker.com/images/oracle-database-enterprise-edition) or the [Oracle Container Registry](https://container-registry.oracle.com) or you can build your own using the Dockerfiles and scripts in [GitHub Location](https://github.com/oracle/docker-images/tree/master/OracleDatabase).
+
+## 3.6. Docker Security Configuration
+
+### A. Configuring auditing rules for docker 
+
+Add the following statements to the audit rules file - `/etc/audit/rules.d/audit.rules`
+
+```
+-w /usr/bin/docker -p wa
+-w /var/lib/docker -p wa
+-w /etc/docker -p wa
+-w /lib/systemd/system/docker.service -p wa
+-w /lib/systemd/system/docker.socket -p wa
+-w /etc/default/docker -p wa
+-w /etc/docker/daemon.json -p wa
+-w /usr/bin/docker-containerd -p wa
+-w /usr/bin/docker-runc -p w
+```
+Restart Audit Daemon using below commands
+```
+sudo systemctl restart auditd
+ ```
+### B. Configuring Docker daemon
+
+Add the following statements to the docker daemon json file - `/etc/docker/daemon.json`
+
+Note: For configuring centralized logging using syslog driver, syslog server details need to be configured. 
+
+Further details can be found at https://docs.docker.com/config/containers/logging/configure/
+```
+{
+    "icc": false,
+    "log-driver": "syslog",
+    "log-opts": {
+        "syslog-address": "Protocol://IpAdressofSysLogserver:port"
+               },
+    "live-restore": true,
+    "userland-proxy": false,
+    "no-new-privileges": true
+}
+```
+Restart docker daemon 
+```
+sudo systemctl daemon-reload
+sudo systemctl restart docker
+sudo systemctl show --property Environment docker
+```
+# 4. Building Oracle WebCenter Content Docker Image
+
+To build/use a WebCenter Content image you should have Oracle FMW Infrastrucure image.
```

Comment:
Just link to the FMW infrastructure README. Don't repeat the instructions. You should remove lines 148-165